iml-system-test-utils/src/lib.rs

```diff
@@ -0,0 +1,68 @@
+pub mod docker;
+pub mod iml;
+pub mod vagrant;
+
+use futures::{future::BoxFuture, FutureExt, TryFutureExt};
+use std::{io, process::ExitStatus, time::Duration};
+use tokio::{process::Command, time::delay_for};
+
+fn handle_status(x: ExitStatus) -> Result<(), io::Error> {
+    if x.success() {
+        Ok(())
+    } else {
+        let err = io::Error::new(
+            io::ErrorKind::Other,
+            format!("process exited with code: {:?}", x.code()),
+        );
+        Err(err)
+    }
+}
+
+pub trait CheckedStatus {
```

Comment:
This exists in the iml-cmd crate. You should be able to reuse that.

Reply:
sure 👍